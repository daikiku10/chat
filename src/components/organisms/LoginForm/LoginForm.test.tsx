/* eslint-disable no-undef */
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import { LoginForm } from ".";

describe("LoginForm", () => {
  it("チャットはじめるボタンが表示されていること", () => {
    render(<LoginForm />);

    const button = screen.getByRole("button");
    expect(button).toBeInTheDocument;
  });

  it("renders homepage unchanged", () => {
    const { container } = render(<LoginForm />);
    expect(container).toMatchSnapshot();
  });
});
