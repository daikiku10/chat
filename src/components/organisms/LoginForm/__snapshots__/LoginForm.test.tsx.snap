// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm renders homepage unchanged 1`] = `
<div>
  <div>
    <form
      action="/chat"
    >
      <div>
        <div
          class="rt-TextFieldRoot"
        >
          <input
            class="rt-TextFieldInput rt-r-size-2 rt-variant-surface"
            maxlength="10"
            name="name"
            placeholder="なまえを入力してください"
            spellcheck="false"
          />
          <div
            class="rt-TextFieldChrome"
          />
        </div>
      </div>
      <div>
        <button
          class="rt-reset rt-BaseButton rt-Button rt-r-size-4 rt-variant-solid"
          type="submit"
        >
          チャットをはじめる
        </button>
      </div>
    </form>
  </div>
</div>
`;
